app/packages/assistant_api/doc/OauthApi.md

```
@@ -0,0 +1,428 @@
+[**metadata**](OauthApi.md#metadata) | **GET** /.well-known/oauth-authorization-server | GET /.well-known/oauth-authorization-server
+[**register**](OauthApi.md#register) | **POST** /oauth/register | POST /oauth/register — register a new OAuth2 client.
+[**revoke**](OauthApi.md#revoke) | **POST** /oauth/revoke | POST /oauth/revoke — revoke a refresh token.
+[**token**](OauthApi.md#token) | **POST** /oauth/token | POST /oauth/token
+
+
+# **authorizeGet**
+> authorizeGet(responseType, clientId, redirectUri, state, codeChallenge, codeChallengeMethod, scope)
+
+GET /oauth/authorize — render login form.
+
+### Example
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)